Library/Homebrew/test/cli_parser_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "../cli_parser"
+
+describe Homebrew::CLI::Parser do
+  describe "test switch options" do
+    subject(:parser) {
+      described_class.new do
+        switch "-v", "--verbose", "Flag for verbosity"
+        switch "--more-verbose",  "Flag for higher verbosity"
+      end
+    }
+    it "parses a single valid option" do
+      args = parser.parse(["--verbose"])
+      expect(args.verbose?).to be true
```

Comment:
`expect(args).to be_verbose` reads nicer IMO

Reply:
Agree. 👍 Updated it accordingly.